go/vt/wrangler/materializer_env_test.go

```diff
@@ -169,11 +172,27 @@ func newTestMaterializerTMClient() *testMaterializerTMClient {
 }
 
 func (tmc *testMaterializerTMClient) GetSchema(ctx context.Context, tablet *topodatapb.Tablet, tables, excludeTables []string, includeViews bool) (*tabletmanagerdatapb.SchemaDefinition, error) {
-	key := tablet.Keyspace + "." + tables[0]
-	if tmc.schema[key] == nil {
-		return &tabletmanagerdatapb.SchemaDefinition{}, nil
+	schemaDefn := &tabletmanagerdatapb.SchemaDefinition{}
```

Comment:
This change is not needed. The lower level function already understands regular expressions.

Reply:
what lower level fn? this is for tests, and will fail w/o this particular fix.

otherwise, there won't be any TableDefs returned for this GetSchema.

Reply:
Sorry. Didn't realize this was  the test function.